space-concordia-robotics/robotics-prototype
# Code linters 416

# Assignee Section

## Description
Changed the README file to include instructions as to how to set up VS Code.
Changed various files used by the formatters and VS Code

### Steps for Testing
Verify that the plugins are appropriate and that their configs are adequate:
- standardjs (no config)
- clangd (.clang-format)
- Python (.yapf    .pylintrc)

Verify that all atom packages are relevant and use the appropriate configs..
Look in `.atom/package-list.txt` for the package names
and `.atom/config.cson` for the custom configs

closes #416 

The approval from all software team leads is necessary before merging.

# Reviewer Section

Aside from local testing and the General Integration Test it is implied that static analysis should be included in the verification process.


For Pull Requests that do not include code changes, it is not required to perform the tests above.
